README.md

```diff
@@ -1,167 +1,140 @@
- 
-## Setup
- 
-Log into OCM, then go here to obtain an [OpenShift Offline Token].
- 
-A properly setup Go workspace using **Go 1.18+ is required**.
- 
-Install dependencies:
-```
-# Install dependencies
-$ go mod tidy
-```
- 
-Set OCM_TOKEN environment variable:
-```
-$ export OCM_TOKEN=<token from step 1>
-```
- 
-## The `osde2e` command
- 
-The `osde2e` command is the root command that executes all functionality within the osde2e repo through a number of subcommands.
- 
-### Running from source
- 
-To run osde2e locally, first build the binary (do this after all changes) by running `make build`. The resulting binaries will be in `./out/`.
- 
-Once built, you can invoke osde2e by running `./out/osde2e`.
- 
-A common workflow is having a local script that combines these steps and the config. Example:
- 
-```bash
-#!/usr/bin/env bash
-make build
- 
-GINKGO_SKIP="" \
-CLEAN_CHECK_RUNS="3" \
-POLLING_TIMEOUT="5" \
-OCM_TOKEN="[OCM token here]" \
-./out/osde2e test --configs "prod,e2e-suite"
-```
- 
-Another example:
-```bash
-#!/usr/bin/env bash
+
+## Introduction
+
+A comprehensive test framework used for Service Delivery to test all aspects of
+Managed OpenShift Clusters ([OpenShift Dedicated]). The data generated by
+the different test coverage is used to inform product releases and decisions.
+
+OSDe2e key features are:
+
+* Portable test framework that can run anywhere to validate end to end test workflows
+  * Run locally from a developers workstation or from a CI application
+* Supports create/delete different cluster deployment types
+  * ROSA, ROSA Hosted Control Plane (e.g. HyperShift), OSD
+* Performs cluster health checks to ensure cluster is operational prior to
+  running tests
+* Perform cluster upgrades
+* Captures artifacts such as logs, metrics, metadata to be archived for later usage
+* Tests OSD operators along with other OpenShift features from a
+  customer/SRE point of view
+* Provides a test harness to validate [Add Ons][OSDE2E Test Harness]
+
+When osde2e is invoked, the standard workflow is followed:
+
+* Load configuration
+* Cluster deployment (when not leveraging an existing cluster)
+* Verify the health of the cluster
+* Run tests (pre upgrade)
+* Collect logs, metrics and metadata
+* Upgrade cluster (when defined)
+* Verify the health of the cluster post upgrade
+* Run tests (post upgrade - when upgrade is defined)
+* Collect logs, metrics and metadata
+* Cluster deprovision (when this is toggled on)
+
+## Prerequisites
+
+Prior to running osde2e, make sure you meet the minimal prerequisites defined below:
+
+* Navigate to [OpenShift Cluster Manager (OCM)][OpenShift Offline Token] to obtain
+  an OpenShift offline token.
+  * Save your token into the environment variable `OCM_TOKEN` for later usage
+* Verify (submit a request if required) your Red Hat account has adequate quota for
```

Comment:
Should we add a reference to https://gitlab.cee.redhat.com/service/ocm-resources/

Reply:
I can, it would make it easier for the user to know exactly where to go. I did not add it initially due to it being an internal repo/this a public project. Though I have seen other repos in OpenShift org that put pointers to that repo. It would not be accessible anyways unless on VPN.